src/xdp/trn_transit_xdp_maps.h

```diff
@@ -40,6 +41,24 @@ struct bpf_map_def SEC("maps") jmp_table = {
 };
 BPF_ANNOTATE_KV_PAIR(jmp_table, __u32, __u32);
 
+struct bpf_map_def SEC("maps") dfts_map = {
+	.type = BPF_MAP_TYPE_HASH,
+	.key_size = sizeof(struct zeta_key_t),
+	.value_size = sizeof(struct dft_t),
```

Comment:
if all dfts_map entry has dft zeta_type, why it's part of key? same applies to "interface". I may misunderstood thinking this map is for maglev lookup, then the key should be result of hash and value should be ftn id, or key should be flow matcher, result is ftn id

Reply:
Address this offline with Bin.
The dft is acquired with a zeta_key_t.
This dft struct then contains the actual maglev hash table.